docs/source/modules/benchmark_tutorial.rst

```diff
@@ -338,6 +338,52 @@ For instance, the following is an excerpt from the
         assert ceiling.raw.median('split') == ceiling
         assert ceiling.uncorrected_consistencies.median('split') < ceiling
 
+Benchmark Card Creation
+-----------------------
+
+Please include a :code:`README.md` file along with your benchmark to aid users with understanding and implementation.
+As part of your README.md file, please include a YAML section using the following format as a guideline:
+
+.. code-block:: python
+
+      benchmark_details:
+        name: <name of the benchmark>
+        developer: <developing individual or organization>
```

Comment:
```suggestion
        developer: <developing individual or group>
```
organization implies e.g. MIT, but it will often make sense on a lab level. To me, "group" sounds agnostic to that

Reply:
done.